Sources/SwiftUIKit/Views/Map.swift

```diff
@@ -0,0 +1,486 @@
+//
+//  Map.swift
+//  SwiftUIKit
+//
+//  Created by Oskar on 12/04/2020.
+//
+
+import Foundation
+import MapKit
+
+public struct MapPoint {
+  let latitude: Double
+  let longitude: Double
+  let title: String
+  let subtitle: String
+}
```

Comment:
`'MapPoint' initializer is inaccessible due to 'internal' protection level`

Swift defaults to internal and not public

```
public struct MapPoint {
  public let latitude: Double
  public let longitude: Double
  public let title: String
  public let subtitle: String
}
```

Reply:
Found in documentation that even when [objects and whole struct are set as `public`](https://github.com/0xLeif/SwiftUIKit/pull/132/commits/575d576ca221c3bf98988038620aeb6764a2870c), its initializer is still `internal`, to change that I had to implement own, [explicitly marked as `public` initializer](https://github.com/0xLeif/SwiftUIKit/pull/132/commits/a9e49c7313e2378b26d6898d749a675a54a6db10).